build/CodeSign/certificate.ignore

```diff
@@ -3,6 +3,9 @@ obj\*, Ignore Intermediates
 bin\*\instrumentationengine.lib.tests.dll, Ignore test assembly
 bin\*\instrumentationengine.profilerproxy.tests.dll, Ignore test assembly
 bin\*\instrenginetests_anycpu.dll, Ignore test assembly
+bin\*\instrenginetests.dll, Ignore test assembly
+bin\*\testapprunner.dll, Ignore test assembly
+bin\*\testapprunner.exe, Ignore test assembly
```

Comment:
I'll have to check, but we might be able to get rid of this apphost during build.

Reply:
Hmm, seems it's needed for the netcore3.1 tests

Reply:
Should not be needed since the test code used the dll not the exe: https://github.com/microsoft/CLRInstrumentationEngine/blob/584c9f7bd3b66f774c527e6a31584fca9dcb09d4/tests/InstrEngineTests/InstrEngineTests/ProfilerHelpers.cs#L204

Reply:
I agree, but for some reason removing the apphost breaks the build: https://dev.azure.com/ms/CLRInstrumentationEngine/_build/results?buildId=146483&view=logs&j=eb496c93-f86e-53a7-8d99-a19210289e6d&t=b9d4457c-6a08-54a4-58a2-8ae830dc4b33

I'll try to investigate this later